conf/etc/v2/ha.conf.template

```diff
@@ -22,6 +22,10 @@ EVENT_MANAGER:
     message_type: health_events
     producer_id: system_health
     consumer_group: health_monitor
+ACTUATOR_MANAGER:
+    req_message_type: requests
+    resp_message_type: alerts
+    producer_id: 1
     consumer_id: 1
```

Comment:
consumer_id should be present in both EVENT_MANAGER and ACTUATOR_MANAGER

Its added to ACTUATOR_MANAGER in https://github.com/Seagate/cortx-ha/pull/538/files 
and is seen for EVENT_MANAGER in https://github.com/Seagate/cortx-ha/blob/main/conf/etc/v2/ha.conf.template

Reply:
@ArchanaLimaye Please review it here https://github.com/Seagate/cortx-ha/pull/563